packages/atlas/src/components/_overlays/SendTransferDialogs/SendFundsDialog.tsx

```diff
@@ -25,12 +25,14 @@ import { hapiBnToTokenNumber, tokenNumberToHapiBn } from '@/joystream-lib/utils'
 import { useMemberAvatar } from '@/providers/assets/assets.hooks'
 import { useFee, useJoystream, useTokenPrice } from '@/providers/joystream/joystream.hooks'
 import { useTransaction } from '@/providers/transactions/transactions.hooks'
+import { formatJoystreamAddress, shortenAddress } from '@/utils/address'
 import { SentryLogger } from '@/utils/logs'
 import { formatNumber } from '@/utils/number'
 
 import { FormFieldsWrapper, PriceWrapper, StyledMaxButton, VerticallyCenteredDiv } from './SendTransferDialogs.styles'
 
-const ADDRESS_LENGTH = 49
+const POLKADOT_ADDRESS_LENGTH = 48
+const JOYSTREAM_FIRST_CHAR = 'j4'
```

Comment:
I'm not comfortable with hardcoding this here - this prefix depends on `JOYSTREAM_SS58_PREFIX` which is exported from `@/joystream-lib/config`. It would be good to keep this config centralized.

Having said that, we will need to know those first chars because they are used in the tooltip copy. However, I think we can easily get those based on that config const. For example:

```ts
const EXAMPLE_ADDRESS = '5Dbstm8wPgrKAwHeMe8xxqxDXyFmP3jyzYdmsiiwTdCdt9iU'
const formattedExampleAddress = formatJoystreamAddress(EXAMPLE_ADDRESS)
const joystreamAddressPrefix = formattedExampleAddress.slice(0, 2)
```